cmd/kyma/provision/minikube/cfg_darwin.go

```diff
@@ -2,4 +2,28 @@
 
 package minikube
 
+import (
+	"fmt"
+	"strings"
+
+	"github.com/kyma-project/cli/internal/cli"
+	"github.com/spf13/cobra"
+)
+
 const defaultVMDriver = vmDriverHyperkit
+
+func osSpecificFlags(o *Options, cmd *cobra.Command) {
```

Comment:
OS specific flags is no longer needed, since the flag is declared in all platforms.